cmd/demo/node.go

```diff
@@ -122,6 +126,17 @@ func importAccount(secret string) (*ewallet.Wallet, *ewallet.Account, error) {
 	return wallet, acc.(*ewallet.Account), err
 }
 
+func getOnChainBal(addressList []wallet.Address) (balList []*big.Int, err error) {
```

Comment:
If you use `addressList... wallet.Address` here the usage becomes easier.

Reply:
Addressed in this commit - 85c0766.